### ISSUE TITLE
Pydantic missing field error on collection_cluster_info response

### ISSUE THREAD

**Issue by ArthurMelin** [qdrant/qdrant-client#771]
Hi, I'm getting a Pydantic missing field exception on the field `obj.result.resharding_operations` when using `qdrant.http.cluster_api.collection_cluster_info(...)`.

When looking at the server source, it looks like currently this field doesn't get serialized if it's empty: https://github.com/qdrant/qdrant/blob/9fa86106e85d734765efe2c686a3ce975e53fb38/lib/collection/src/operations/types.rs#L210

Environment:
qdrant-client: 1.11.1
qdrant: 1.11.3

Traceback:
```
Traceback (most recent call last):
  File "/data/scripts/rebalance.py", line 10, in <module>
    cluster_info = qdrant.http.cluster_api.collection_cluster_info(collection).result
  File "/data/scripts/venv/lib64/python3.9/site-packages/qdrant_client/http/api/cluster_api.py", line 302, in collection_cluster_info
    return self._build_for_collection_cluster_info(
  File "/data/scripts/venv/lib64/python3.9/site-packages/qdrant_client/http/api/cluster_api.py", line 81, in _build_for_collection_cluster_info
    return self.api_client.request(
  File "/data/scripts/venv/lib64/python3.9/site-packages/qdrant_client/http/api_client.py", line 79, in request
    return self.send(request, type_)
  File "/data/scripts/venv/lib64/python3.9/site-packages/qdrant_client/http/api_client.py", line 101, in send
    raise ResponseHandlingException(e)
qdrant_client.http.exceptions.ResponseHandlingException: 1 validation error for ParsingModel[InlineResponse2008] (for parse_as_type)
obj.result.resharding_operations
  Field required [type=missing, input_value={'peer_id': 4660132199939..., 'shard_transfers': []}, input_type=dict]
    For further information visit https://errors.pydantic.dev/2.9/v/missing
```

**Comment by joein** [comment]
Hi @ArthurMelin 

Could you provide a minimal reproducible code snippet please?
So we could confirm the problem and also track the way you're using it

**Comment by ArthurMelin** [comment]
Sure, here it is:
```python
from qdrant_client import QdrantClient

qdrant = QdrantClient()

peers = [p for p in qdrant.http.cluster_api.cluster_status().result.peers]

collections = [c.name for c in qdrant.http.collections_api.get_collections().result.collections]

for collection in collections:
    cluster_info = qdrant.http.cluster_api.collection_cluster_info(collection).result
    target_shards_per_peer = cluster_info.shard_count / len(peers)
```
I haven't finished writing the script so I can't provide more than this but its purpose is to automatically start shard transfers to rebalance the cluster after a new peer has been added.

**Comment by joein** [comment]
Hi @ArthurMelin 
It seems to be a bug in our openapi schema, which resulted in generating the incorrect model.
Thanks for pointing it out

**Comment by joein** [comment]
Hey @ArthurMelin 

The fix should be available with the next release
You can try it out now with the dev branch

**Comment by joein** [comment]
Should be available as of v1.11.3
